frictionless/resources/table.py

```diff
@@ -310,7 +310,18 @@ def row_stream():
             for row_number, cells in enumerated_content_stream:
                 self.stats.rows += 1
 
-                # Create row
+                # NB: missing required labels are not included in the
```

Comment:
Is it possible to move it from the row iteration cycle? Currently, this code will be executed for every row in a table

Reply:
Thanks!
Indeed, you're right, it's possible to move this block before initializing `row_stream` `TableResource` attribute.
I've just committed it. I also added some new test cases to insure that resulting behaviour corresponds to the expected one.